src/parser/warlock/affliction/modules/talents/GrimoireOfSacrifice.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import { formatThousands } from 'common/format';
+
+import Analyzer from 'parser/core/Analyzer';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+
+import SpellLink from 'common/SpellLink';
+import StatisticListBoxItem from 'interface/others/StatisticListBoxItem';
+
+class GrimoireOfSacrifice extends Analyzer {
```

Comment:
If you were to die, does GrimoireOfSacrifice need to be recasted? If so should there be a suggestion for recasting it?